blocks/contact-form/contact-form.js

```diff
@@ -1,15 +1,27 @@
-// Selectors
+import submitContactInfo from './submit-contact-info.js';
+
+// Selector for field divs
 const blockChildDivs = document.querySelectorAll('.contact-form > div');
 
 export default function decorate(block) {
-  // Create submit button
-  const submitBtn = document.createElement('button');
-  submitBtn.textContent = 'Submit';
-  submitBtn.classList.add('contact-form__button');
-  // Add submit button
-  block.appendChild(submitBtn);
-
-  // Create input elements for each field
+  // Create elements
+  const form = document.createElement('form');
+  const submitBtnDiv = document.createElement('div');
+  const submitBtnLink = document.createElement('a');
```

Comment:
Why use an anchor element for the submit? There's two valid html alternatives, `button` and an `input[type=submit]`
I'd prefer those unless you have an explicit reason for the anchor